### ISSUE TITLE
Some issues/questions regarding Matrix/Synapse

### ISSUE THREAD

**Issue by ArchangeGabriel** [matrix-org/synapse#2031]
Hi there,

I’m new to matrix, and trying to setup everything. I had some troubles with different parts of the process (especially, I wasn’t sure whether I should regenerate certs for the subdomain in the SRV record or use the one for the main domain, apparently it is the latter), but finally ended up with a working configuration.

An other issue I have is user registration from the README doesn’t work:
```
New user localpart: archange
Password: 
Confirm password: 
Make admin [no]: yes
Sending registration request...
ERROR! Received 404 Not Found
```
(BTW, I don’t know what the make admin is for)

So I temporarily enabled user registration and registered through riot-android, which worked (but I suppose then that I’m not admin, though I still don’t know what that means).

(Note: After deleting the database and restarting this creation, I’ve figured out that this is because it tries to connect to https://localhost:8448 where client API is not present/deactivated — asking for http://localhost:8008 instead worked)

I’ve then tried to join #matrix-dev:matrix.org as suggested, and had a hard time figuring that I should first create an empty room before being able to send the join command. It eventually worked, and here begin the second issue: after leaving the room, my HS apparently still retains info from this room and receive update/store info from it, which I don’t want. Is there a way to purge all the related data from the DB and stop receiving update from federated servers taking part in this discussion?

I’ve tried removing the DB and restarting the server (also because I wanted to reset my IRC configuration), but it didn’t worked apparently — which is kind of expected I guess since other HS still retains the information that I was part of the discussion at some point.

In the same vein, if I (or the IRC appservice) create room on my HS and then leave them, they’re still floating around. How can I remove them and all the related data?

Also, something that is not very clear to me: if I create rooms on my HS, can people from other HS join them? I suppose so, but only if such rooms are federated, right (though I don’t know how I can make non-federated rooms excepted apparently for the IRC bridge)? What happens if someone tries to join #someroom:myhomeserver.tld?

Thanks for your help!

**Comment by majewsky** [comment]
I'm also new to all of this, but regarding joining existing channels from other servers, what worked for me (in Riot for Andriod) is to use the search button on the main view, and type in a channel URI like `#matrix-dev:matrix.org`. Once you spell it all out, it'll offer to join that channel. The UI was also a bit weird: I tried it with `#matrix:matrix.org` and it showed a spinner for a really long time as if the operation was still in progress. When I manually aborted it and went back to the main view, the channel was listed as joined and everything worked.
